frontend/src/helper.js

```diff
@@ -130,23 +134,26 @@ export const PROFILE = {
     faIcon: 'shield',
     hexcolor: '#ed0007',
     tradKey: i18n.t('Administrator'), // trad key allow the parser to generate an entry in the json file
-    label: 'Administrator' // label must be used in components
+    label: 'Administrator', // label must be used in components
+    description: 'Create user, create shared space, administration of instance'
```

Comment:
without the translation function, the script `npm run build-translation` wont create any translation.
We need to have a mechanism that ensure that the above script will generate the keys properly.
We will need to discuss about that